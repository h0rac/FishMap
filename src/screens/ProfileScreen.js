import React, {Component} from 'react';
import {Text,View,Image, TouchableHighlight, Platform} from 'react-native'
import {connect} from 'react-redux';

export class ProfileScreen extends Component {

    static navigationOptions = {
        title: "Profile",
        headerTintColor: '#2F95D6',
    }

    render() {
        return <Text>Hello in Profile!</Text>;
    }
}

export default connect()(ProfileScreen)
