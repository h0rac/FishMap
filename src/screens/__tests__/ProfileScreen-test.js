import 'react-native';
import React from 'react';
import { ProfileScreen } from '../ProfileScreen';
import renderer from 'react-test-renderer';


it('renders correctly', () => {
	const tree = renderer.create(<ProfileScreen/>);
	expect(tree).toMatchSnapshot();
});