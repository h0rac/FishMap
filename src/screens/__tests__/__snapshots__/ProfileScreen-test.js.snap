// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  disabled={false}
  ellipsizeMode="tail"
>
  Hello in Profile!
</Text>
`;
